### ISSUE TITLE
Visually labelling unique and legendary items / prevent them to be sold

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#214]
```
mattbernius told here:
http://andors.techby2guys.com/viewtopic.php?f=4&t=620

It would be great if -- in the inventory -- unique/legendary items (like the 
Venomous Dagger, Flagstone Pride, the Ring of Lesser Shadow) had some type of 
visual cue or entry on their "description" screen that signals that they are a 
unique item.

qasur told here:
http://andors.techby2guys.com/viewtopic.php?f=4&t=844
Can we get an "Are you sure?" button when you're about to sell boss items?

Perhaps we should also disable selling of this items at all.
```

Original issue reported on code.google.com by `SamuelPl...@gmail.com` on 9 Jul 2011 at 7:18

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the report. Please try to separate different requests into separate 
issues.

The first request (visually labelling different types of items) is now commited 
in r132. 

In response to the second request, I guess this type of indication combined 
with the quantity dialog (from issue 59) should work as sufficient indication 
to the player.

Marking this issue as ready for the next release.
```

Original comment by `oskar.wi...@gmail.com` on 16 Jul 2011 at 10:11
- Changed state: **Ready**
- Added labels: **Type-Enhancement**, **Milestone-0.6.10**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.10 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 21 Oct 2011 at 8:56
- Changed state: **Fixed**
